packages/babel-helper-builder-react-jsx/src/index.js

```diff
@@ -28,6 +29,20 @@ export default function(opts) {
     },
   };
 
+  visitor.JSXFragment = {
+    exit(path, file) {
+      if (opts.compat) {
+        throw path.buildCodeFrameError(
+          "Fragment tags are only supported in React 16 and up.",
```

Comment:
Should we do a peerDep on react @babel/react? (maybe for the preset)

Reply:
I think `Fragment` name is also only exported in `16.1`, even though it's technically supported in `16.0`

Reply:
What do you mean by a peerDep on react @babel/react?

Reply:
I mean should `@babel/preset-react` have a peerDep on `react or react-dom`?

Reply:
Hmm, good question. Would non-React library authors use `@babel/preset-react`? It seems like enabling `@babel/preset-react` and providing pragmas allows Babel to compile to alternative JSX rendering libraries.

Reply:
Ah yeah they may just use the specific transform itself